### PR TITLE
docs:Update readme

### PR DESCRIPTION
**Pull request by soichiro-hattori**
Starting a draft PR to address #122 and update the `README.md` file. To avoid having to maintain two duplicates of an intro doc, the `README.md` will be the file with content and `index.md` will simply keep track of the `README.md` file.

### CODE REVIEW

**Comment by soichiro-hattori** [comment]
@dfm: Here's a draft version of the new README/RTD landing page!
I'm not entirely sure, but I think we might need to think about using absolute file paths when creating references since the `README.md` & `index.md` file exist in different directories?
Also, would the toctree render in the README on Github?

**Review by dfm — CHANGES_REQUESTED** [review]
Looks great! One tiny comment then I think we should merge.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,75 @@
...
+installation.md
+start
+tutorials/autodiff.ipynb
+tutorials/orbits.ipynb
+tutorials/transit.ipynb
+tutorials/core-from-scratch.ipynb
+tutorials/rv.ipynb
+tutorials/starry.ipynb
+api
+```
```
[the remark this answers is not in the document]
Comment (soichiro-hattori) [reply README.md]: Great that's what I was also thinking!

**Comment by soichiro-hattori** [comment]
@dfm: There still isn't any text in the "Quick Start" part of the `README.md` file but otherwise I think it's ready to go for now!

**Pull request merged**